### DISCUSSION TITLE
Freeze SymbiotaID column on the Data Editor Search Form

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#592] [Wish List]
Multiple CCH2 users would like the _SymbiotaID_ column to be frozen to facilitate the selection/opening of records while browsing search results:
![Screenshot 2024-08-20 at 2 45 44 PM](https://github.com/user-attachments/assets/9166a6e3-945f-4184-afd0-b249342fa25c)

Similar to: https://github.com/BioKIC/symbiota-docs/issues/525

**Comment by umn-annakfowler** [comment]
Does the top row (column headers) also freeze when you scroll down?

**Reply by ljwalker** [reply]
No, but that enhancement request has also been documented: https://github.com/BioKIC/symbiota-docs/issues/525

**Reply by umn-annakfowler** [reply]
Great, thanks Lindsay!
